app/views/home/volunteer.html.erb

```diff
@@ -0,0 +1 @@
+this is a volunteer

```

Comment:
I don't think their old website has a volunteer page.  We should leave this out.  Or you can change it to a contact us page; we need that.

Reply:
I based the links off <a href="https://docs.google.com/document/d/11qyPjdsJG0jgQcOInD63jAH33J_eZq7cgTBXAtVrWL8/edit">this document</a> she shared in the group chat.

Reply:
I see.  Maybe rename it or clarify that it is "get involved" and provides more info than just volunteering like mailing list and/or contact us.

Reply:
named it ```participate``` mostly because i didn't want to deal with the space